frontend/components/events/EventCard.tsx

```diff
@@ -39,9 +38,12 @@ export default function EventCard(props: EventCardProps) {
           <>
             <div
               className="h-36 w-full object-cover rounded-t-lg"
-            >
-              <EventImage imageSrc={props.image} />
-            </div>
+              style={{
```

Comment:
could we abstract this div to an image component.
Also, what was wrong with the old EventImage component? Was that the one causing the resizing issues?

Reply:
yeah it was

Reply:
unfortunately using a `next/image` affects our loading performance on the dashboard which is weird. Using it as a `backgroundImage` some how seems to tell the browser to prioritise the load and not lazy load it, which seems to be faster on user testing..